doc/modules/model_evaluation.rst

```diff
@@ -1075,6 +1075,36 @@ Here is a small example of usage of this function::
     >>> label_ranking_average_precision_score(y_true, y_score) # doctest: +ELLIPSIS
     0.416...
 
+Ranking loss
+------------
+
+The :func:`label_ranking_loss` function computes the ranking loss which
+averages over the samples the number of label pairs that are incorrectly
+ordered, i.e. true labels have a lower score than false labels, weighted by the
```

Comment:
This is seems to contradict the doctest examples. Based on the doctest, it seems to be "... incorrectly
+ordered, i.e. true labels have a **higher** score than false labels". Although I find that counter-intuitive.